mplc/scenario.py

```diff
@@ -304,13 +304,21 @@ def __init__(
 
         now = datetime.datetime.now()
         now_str = now.strftime("%Y-%m-%d_%Hh%M")
-        self.scenario_name = f"scenario_{self.scenario_id}_repeat_{self.n_repeat}_{now_str}_" \
-                             f"{uuid.uuid4().hex[:3]}"  # to distinguish identical names
-        self.short_scenario_name = f"{self.partners_count} {self.amounts_per_partner}"
-
-        self.experiment_path = experiment_path
-        self.save_folder = self.experiment_path / self.scenario_name
-
+        self.scenario_name = kwargs.get('scenario_name',
+                                        f"scenario_{self.scenario_id}_repeat_{self.repeat_count}_{now_str}_"
+                                        f"{uuid.uuid4().hex[:3]}")  # to distinguish identical names
+        if ' ' in self.scenario_name:
```

Comment:
Doesn't seem robust - I'd suggest regex in the form of `r'\s'` or something like that